src/System.Private.DataContractSerialization/src/System/Xml/XmlUTF8TextWriter.cs

```diff
@@ -5,6 +5,7 @@
 using System.Runtime;
```

Comment:
Why is there a private copy of XML here? Is there something we should add to System.Xml.* to remove it?